src/audio/pipeline.c

```diff
@@ -771,9 +770,12 @@ static int pipeline_copy_from_upstream(struct comp_dev *start,
 		buffer = container_of(clist, struct comp_buffer, sink_list);
 
 		/* don't go upstream if this component is not connected */
-		if (!buffer->connected || buffer->source->state != COMP_STATE_ACTIVE)
+		if (!buffer->connected)
```

Comment:
This will breal pipelines with a mixer, where closing one source stream will probably stop the sink stream (even if other source streams are active )?